src/main/java/uk/gov/hmcts/sptribs/document/model/CICDocument.java

```diff
@@ -43,7 +43,7 @@ public CICDocument(@JsonProperty("documentEmailContent") String documentEmailCon
 
     @JsonIgnore
     public boolean isDocumentValid() {
-        String regex = ".pdf,.tif,.tiff,.jpg,.jpeg,.png,.mp3";
+        String regex = ".pdf,.tif,.tiff,.jpg,.jpeg,.png,.mp3,.m4a";
```

Comment:
Good time to create a respective test file to cover the unit testing?

Reply:
Will create one